packages/ckeditor5-table/src/tableediting.js

```diff
@@ -120,6 +120,10 @@ export default class TableEditing extends Plugin {
 		// Table heading columns conversion (a change of heading rows requires a reconversion of the whole table).
 		conversion.for( 'editingDowncast' ).add( downcastTableHeadingColumnsChange() );
 
+		// Manually adjust model position mappings in a special case, when a table cell contains a paragraph, which is bound
+		// to its parent (to the table cell).
+		editor.data.mapper.on( 'modelToViewPosition', mapTableCellModelPositionToView( editor.editing.view ) );
```

Comment:
Please add a comment on why this is needed only in the data pipeline.

Reply:
Added.